h/templates/layouts/base.html.jinja2

```diff
@@ -81,6 +81,10 @@
       </script>
     {% endif %}
 
+    <script>
+    {% include "h:../build/scripts/header.bundle.js" %}
```

Comment:
This isn't great if we're trying to make the site CSP-safe. I get why you've inlined this but I'd suggest that for now we load it with a `<script src>`.